Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -995,5 +995,7 @@ public bool HasMachineId(byte[] bytes, int machineId)
                     return MachineIdSet.HasMachineId(reader, localIndex);
                 });
         }
+
+        internal abstract Result<long> GetContentDatabaseSizeBytes();

```

Comment:
A comment? And why it is internal and not public? Just curious, because we usually using internal members because they're used in tests only...